src/lamp_py/tableau/jobs/bus_performance.py

```diff
@@ -76,7 +78,29 @@ def create_bus_parquet(job: HyperJob, num_files: Optional[int]) -> None:
 
     with pq.ParquetWriter(job.local_parquet_path, schema=job.parquet_schema) as writer:
         for batch in ds.to_batches(batch_size=500_000):
-            writer.write_batch(batch)
+            arrow_table = pyarrow.Table.from_batches([batch])
+
+            polars_df = pl.from_arrow(arrow_table)
```

Comment:
`polars.from_arrow` can read in a RecordBatch or Table, so `batch` can be passed directly into this function and your `arrow_table` assignment can be removed.